tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/errors/BigQuerySqlError.java

```diff
@@ -0,0 +1,21 @@
+package com.google.cloud.bigquery.utils.queryfixer.errors;
+
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.utils.queryfixer.entity.Position;
+
+import lombok.Getter;
+
+/** The base class to represent the SQL Error generated by the BigQuery server. It should be resolved from
+ * {@link BigQueryException}. */
```

Comment:
nit: not sure why the formatter starts new line again ==

Reply:
Ah, I think the first line is almost 100 chars, so it should be a new line.